epub-a11y-meta-guide/1.0/draft/index.html

```diff
@@ -1379,58 +1379,614 @@ <h5>Sign Language</h5>
+                        </pre>
 
-					<section id="metadata-a11y-features-MathML">
-						<h5>Math Represented using MathML</h5>
+                        
+                        <div class="note">
+                           <p><em>The above MathML structure is verbose. A more common authoring method is to use a tool or converter that generates the correct MathML from a simpler input.</em></p>
 
-					</section>
+                            <p><em><strong>Future-facing Note (MathML 4):</strong> The upcoming MathML 4 specification, currently in development by the W3C as of June 2025, introduces an <code>intent</code> attribute to explicitly declare the semantic meaning of a formula. This will allow authors to unambiguously state that a formula represents chemistry, greatly improving accessibility and machine-readability. An example might look like this:</em></p>
+                            <pre>
+                                <code>&lt;math xmlns="http://www.w3.org/1998/Math/MathML" intent="chemical-formula('water')"&gt;
+                      &lt;!-- ... MathML for H2O ... --&gt;
+                    &lt;/math&gt;</code>
+                            </pre>
+                        </div>
 
-					<section id="metadata-a11y-features-ttsMarkup">
-						<h5>Text-To-Speech Markup</h5>
 
-					</section>
+                        <p>The metadata to include in the package document to advertise this feature would be:</p>
+                        <pre>
+                            <code>&lt;meta property="schema:accessibilityFeature"&gt;MathML-chemistry&lt;/meta&gt;</code>
+                        </pre>
+                    </aside>
+                </section>
+
+				<section id="metadata-a11y-features-latex">
+                    <h5>Math represented using LaTeX</h5>
+
+                    <p>Identifies that the LaTeX typesetting system is used to encode mathematical or other scientific notations.</p>
+                    <p>In this method, the raw LaTeX source is embedded directly in the content document. A JavaScript-based rendering engine, such as MathJax, should then be used to convert this source into a visually and accessibly rendered formula.</p>
+
+                    <aside class="example" title="AccessibilityFeature: latex">
+                        <p>The core of this method is the raw LaTeX source code embedded in the text. The example below shows the LaTeX for a common mathematical formula.</p>
+
+                        <p><strong>1. Raw LaTeX Data in HTML:</strong></p>
+                        <pre>
+                            <code>&lt;p&gt;The quadratic formula is \[x = \frac{-b \pm \sqrt{b^2 - 4ac}}{2a}\].&lt;/p&gt;</code>
+                        </pre>
+
+                        <p><strong>Important:</strong> The example above is incomplete on its own. Without a rendering engine, it would only display the raw text of the LaTeX source. To be functional, a library must be included to process it.</p>
+
```

Comment:
It'd likely be easier to read these as separate examples - i.e., break the aside here and start a new one.